django/db/models/base.py

```diff
@@ -335,12 +338,9 @@ def make_foreign_order_accessors(field, model, cls):
                     'set_%s_order' % cls.__name__.lower(),
                     curry(method_set_order, cls)
                 )
-            add_lazy_relation(
-                cls,
-                opts.order_with_respect_to,
-                opts.order_with_respect_to.rel.to,
-                make_foreign_order_accessors
-            )
+            lazy_related_operation(make_foreign_order_accessors,
```

Comment:
@timgraham can correct me if I'm wrong, but I think we prefer an indentation style where all indents are always exactly four spaces (as in the code you've replaced here), not a hanging indent at an arbitrary level (as in the code you've introduced).